jax/_src/pjit.py

```diff
@@ -484,6 +517,7 @@ def _parse_jit_arguments(fun: Callable, in_shardings: Any, out_shardings: Any,
         use_resource_env=use_resource_env,
         compiler_options_kvs=compiler_options_kvs)
 
+# DO_NOT_SUBMIT: remove this function
```

Comment:
Can you remove the functions below? (For some reason, I can't comment on those lines lol)

Reply:
Of course, that is why I left the DO_NOT_SUBMIT. I want to keep this code until I debug all the google internal test failures.